### ISSUE TITLE
"Missing" filter doesn't show all the missing movies

### ISSUE THREAD

**Issue by alexandrevryghem** [Radarr/Radarr#4626]
**Describe the bug:**
On the home page you can filter the movies with: All, Monitored/Unmonitored, Missing, Wanted, ... . The missing filter doesn't display the missing movies that are Unmonitored.

**Expected behavior:**
The missing filter should show all the movies (Monitored AND Unmonitored) that aren't downloaded and the Wanted filter should only show the Monitored movies that aren't downloaded.

For the moment both filters show the same thing (all the Monitored movies that aren't downloaded). 
So the missing filter should also show the Unmonitored not downloaded movies

[AB#863](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/863)

**Comment by austinwbest** [comment]
That is by design because they are not monitored so therefor not wanting to be seen as missing. They are to be ignored. What you would need to do is simply create a custom filter. Maybe try where the size on disk equals 0 or something like that and name it All Missing..

**Comment by r0747249** [comment]
Ok, sorry I thought it was a bug because on sonarr the Missing Episodes filter shows the missing episodes for the Monitored and Unmonitored series

**Comment by austinwbest** [comment]
No need to apologize, I'm just giving what I think it should be based on the naming. It may very well be a bug of some one else interprets it differently.

To me, not monitored means to ignore looking for anything related to it (think that is the actual point of it as well) so with that logic, I wouldn't want to see its missing as it will always be missing, impossible to auto grab with that status. 

Wanted are the things I want which would be anything I am monitoring of course but also things I have set to upgrade to a better quality. 

Again, just my perception based on the meaning of each one individually.

**Comment by austinwbest** [comment]
Ok, so a different thought..

Missing > Anything that is in radarr and does not have a file on disk. Pretty simple and technically is accurate
Wanted > Anything that is (monitored + missing) or (cutoff unmet). Pretty simple and technically is accurate

Currently it seems 

wanted = anything missing that i want
cutoff = anything not missing that i want

Both of those are technically Wanted so put them in the same filter and let the UI differentiate them with the progress bar colors, etc.

**Comment by geogolem** [comment]
I am thinking the best way to fix this.. is to remove the default filters altogether.. implement them as custom filters.. then the user can see or customize what they actually do... The default ones could be added as custom filters perhaps as a migration.. so that nobody would notice any change; however, the user could then see the components, modify or remove filters altogether.

The only issue I see with this, is if the user deletes them, then wants the "default" ones back. They could always easily add them back manually.. so i would just consider that a user error. Otherwise maybe a way to add the "default" ones back more easily.. --> though i think it is easy enough. 

In my opinion its better if the user defines their own filters -- so that they know precisely what they are.

**Comment by bakerboy448** [comment]
@geogolem just reminding you of this one as well

**Comment by psi5asp** [comment]
Sorry to chime in, but if unmonitored are ignored, why are they even listed as "Missing" on the list? The logic doesn't sound right.

Here (unfiltered) they are listed as missing (in my case, mis-matched):
[https://ibb.co/jyhtdcp](url)

Here applying the folder to the same list, the do not show up - they should since filters normally apply to the list the menu option exists on:
[](https://ibb.co/wpRD0Lc)

Could this please be reconsidered? I can understand unmonitored = ignored, but that use of ignore applies to 'upgrading' or 'finding' new versions of the movie, etc. already in the DB, not ignoring them in Radarr DB/library all together (otherwise they'd be removed from the library as 'ignored' - which is what delete does).

**Comment by sentry07** [comment]
The whole "missing & unmonitored means you don't want it" doesn't make sense to me. If I didn't want the movie I wouldn't have added it in Radarr. I just want to see ALL movies that are missing. Unless the movie has not come out yet, I almost never turn on monitoring because I want to search for the movie manually and select a file to download. If I have movies monitored, then Radarr is constantly downloading movies that I already have and I don't want that. 

The filter says "Missing" and the movie is missing. Whether or not it's monitored is my option, but if it's missing, I want to see it. It was obviously designed that way because there are labels for "Missing (Monitored)" and "Missing (Unmonitored)" so why would Missing (Unmonitored) movies not show up when I filter by Missing? I honestly think you're over thinking this. If you want to make the filter only show monitored missing movies, label it that way. And then please give the option in the custom filters for movies that are missing.

**Comment by bakerboy448** [comment]
Size On Disk < 1

Your custom filter is possible.

**Comment by SamKook** [comment]
I have this problem and using a custom filter of Size on Disk < 1 doesn't work for me since that will return all of them in my specific situation.

I have all movies set to unmonitored since I want them all in one folder(which radarr didn't support last I checked) instead of each in their own so I have a script that takes care of all of that once a movie gets added(so radarr only fetches metadata and pics basically). That way, I can keep track of what I have and what's missing since nothing gets updated when I move the movie away somewhere else.

Making the status(missing, wanted, etc.) available for custom filters should be very easy and would fix that. The Missing filter could remain with its current behavior of ignoring unmonitored movies and everybody could achieve what they want.

**Comment by bakerboy448** [comment]
Simply add monitoring status to your existing custom filter.

**Comment by SamKook** [comment]
I have Monitored as false in my custom filter, but as I said, every movie is unmonitored and 0 bytes(not quite but most got refreshed to that) so I currently have no way of simply making the missing unmonitored show up, I have to manually hunt for that yellow bar and watch the whole list manually.

**Comment by bakerboy448** [comment]
There's no solution for you then as if radarr has no movie file due to your unsupported setup and the movie is unmonitored the custom filter logic you're using and the native functionality will be the same.

Please use one of the support channels: [Discord](https://radarr.video/discord) for support/questions.

**Comment by SamKook** [comment]
Status isn't my term, it's what it's identified as when you open a movie detail so that's why I used that term.
I don't see how that can be true since radarr does properly identify which ones of my unmonitored movies are missing with the yellow bar and the missing status if I click on them(even if I refresh them) so it has to have a way to know and there should be no reason that can't be added to the filter list.
I have unmonitored movies with a 0 byte file size that are properly identified as not missing since an empty file exist in their place(which is something I didn't mention and probably why you think it cannot work). If I remove that file and refresh the status, then it will indeed not be able to tell.
But as far as I can tell, it's also not possible to filter if a file exist or not. I tried smaller than 1 byte and not 0 byte, but I'm guessing it considers missing to be the same as 0 bytes in the filter.

It would solve my edge case use and people like the OP who wants the Missing filter to apply to unmonitored movies by making their own custom Missing filter.
